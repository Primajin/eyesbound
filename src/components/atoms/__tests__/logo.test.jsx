import React from 'react';
import {create} from 'react-test-renderer';

import Logo from '../logo.jsx';

describe('Logo', () => {
	it('renders correctly without props', () => {
		const component = create(<Logo/>);
		expect(component).toMatchSnapshot();
	});

	it('renders correctly with props', () => {
		const component = create(<Logo isFullscreen/>);
		expect(component).toMatchSnapshot();
	});
});
