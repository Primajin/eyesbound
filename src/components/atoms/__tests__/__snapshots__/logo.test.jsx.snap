// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly with props 1`] = `
<h1
  className="isFullscreen"
  css={
    {
      "name": "1vze745",
      "next": undefined,
      "styles": "
	background-color: var(--foreground);
	color: var(--background);
	font-size: 27px;
	font-weight: 700;
	height: 100px;
	left: 0;
	line-height: 100px;
	opacity: 1;
	position: fixed;
	text-align: center;
	transition: opacity .333s;
	top: 0;
	width: 250px;
	z-index: 10;

	&.isFullscreen {
		opacity: 0.65;
	}

	@media (min-width: 576px) {
		font-size: 37px;
		width: 400px;
	};
",
    }
  }
>
  <a
    href="/"
    tabIndex="1"
  >
    EYESBOUND
  </a>
</h1>
`;

exports[`Logo renders correctly without props 1`] = `
<h1
  className=""
  css={
    {
      "name": "1vze745",
      "next": undefined,
      "styles": "
	background-color: var(--foreground);
	color: var(--background);
	font-size: 27px;
	font-weight: 700;
	height: 100px;
	left: 0;
	line-height: 100px;
	opacity: 1;
	position: fixed;
	text-align: center;
	transition: opacity .333s;
	top: 0;
	width: 250px;
	z-index: 10;

	&.isFullscreen {
		opacity: 0.65;
	}

	@media (min-width: 576px) {
		font-size: 37px;
		width: 400px;
	};
",
    }
  }
>
  <a
    href="/"
    tabIndex="1"
  >
    EYESBOUND
  </a>
</h1>
`;
